docs/authentication/enterprise-connections/oidc/custom-provider.mdx

```diff
@@ -29,33 +29,48 @@ To make the setup process easier, it's recommended to keep two browser tabs open
 
   1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
   1. Select **Add connection** and select **For specific domains**.
-  1. Under **Third party**, select **OIDC**.
+  1. Under **Third party**, select **OpenID Connect (OIDC)**.
   1. Add the **Name** of the connection.
   1. Add the **Key** of the provider. This is the provider's unique identifier (cannot be changed after creation).
   1. Add the **Specific Domain** that you want to allow this connection for. This is the domain of the users you want to allow to sign in to your app.
   1. Select **Add connection**. You will be redirected to the connection's configuration page.
 
-  The next steps involve configuring both your IdP and Clerk. Keep both dashboards open as you'll need to reference them.
-
   ### Configure your IdP
 
   1. If necessary, create a new app in your IdP.
-  1. In the connection's configuration page in the Clerk Dashboard, copy the **Authorized redirect URI**.
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
+  1. Copy the **Authorized redirect URI**.
   1. Add the value to your IdP's whitelisted URLs.
 
   ### Set the Discovery Endpoint, Client ID, and Client Secret in Clerk
 
-  1. In your IdP settings, copy your application's **Discovery Endpoint**, **Client ID**, and **Client Secret**.
-  1. In the connection's configuration page in the Clerk Dashboard, paste these values in their respective fields.
-  1. Add the minimum required scopes based on the IdP's documentation. Common OIDC scopes include `openid`, `email`, and `profile`.
+  Most IdPs provide a **Discovery Endpoint** to retrieve metadata about an OIDC provider. If your IdP doesn't offer this endpoint or if you need greater control over the setup process, select **Use Manual Configuration** in the **Identity Provider Configuration** section to manually configure the connection.
+
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
```

Comment:
You wouldn't get the client ID and client secret in the clerk dashboard. you would get those from your IdP and then paste them in the clerk dashboard. so i believe the copy before was correct